### PR TITLE
#29 define django model for investors

### PR DESCRIPTION
**Pull request by alona-ruda**

### CODE REVIEW

**Review by avu-ua — APPROVED** [review]

File: ForumProject/investors/models.py [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+class UserInvestor(models.Model):
+    customuser = models.ForeignKey(CustomUser, on_delete=models.CASCADE)
+    investor = models.ForeignKey(Investor, on_delete=models.CASCADE)
+    investor_role_id = models.IntegerField()
+
+
+class InvestorProject(models.Model):
+    investor = models.ForeignKey(Investor, on_delete=models.CASCADE)
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    share = models.IntegerField()
```
Comment (mehalyna) [inline ForumProject/investors/models.py]: Ensure that this aligns with the business logic and consider adding constraints like `models.CheckConstraint` to ensure the values are within a valid range (e.g., 0-100 for percentages)

File: ForumProject/investors/models.py
Comment (mehalyna) [inline ForumProject/investors/models.py:1]: Adding docstrings to your models, especially for methods like `clean`, can provide helpful context for other developers

**Review by mehalyna — CHANGES_REQUESTED** [review]

File: ForumProject/investors/models.py (hunk 1 above)
Comment (alona-ruda) [reply ForumProject/investors/models.py]: I added models.CheckConstraint

File: ForumProject/investors/models.py
Comment (alona-ruda) [reply ForumProject/investors/models.py:1]: Done

**Review by mehalyna — APPROVED** [review]

**Review by DmAlexx — APPROVED** [review]

**Pull request merged**